### PR TITLE
Signup Front End

### PR DESCRIPTION
**Pull request by pmarks98**
Worked on the frontend design of three views: Signup, Login, Splashscreen. 

All three views do not have navigation yet between them. 

I had to go down into some of the lower-level frameworks to get the keyboard hiding working on the signup. SwiftUI has no built-in feature to prevent the keyboard from displaying over the TextField inputs. 

Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 08 27
Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 08 45

Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 09 01

### CODE REVIEW

File: frontend/RaffleBay/RaffleBay/View/SignupView.swift [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+//TO DO:
+//Figure out a way when clicking outside of the form area
+//the keyboard should be dismissed.
+
+
+struct SignupView : View {
+    
+    //I don't know if this is the best/correct way to accept input.
+    //Another way I think could work is like a hashmap that maps inputs to their associated values. Will look into that.
+    @State private var firstName: String = ""
```
Comment (mrachamallu) [inline frontend/RaffleBay/RaffleBay/View/SignupView.swift]: add these variables to the user.swift file in the model file. then make a user object in this class and update it's variables

File: frontend/RaffleBay/RaffleBay/View/SignupView.swift [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                    .opacity(0.6)
+                    
+                    //Signup Button
+                    Button(action: {
+                        
+                    }){
+                        Text("Signup")
+                            .fontWeight(.medium)
+                            .padding(10)
+                            .font(.custom("Poppins", size: 30))
```
Comment (mrachamallu) [inline frontend/RaffleBay/RaffleBay/View/SignupView.swift]: make a style.swift file that contains all the styling constants like "Poppins" etc

**Comment by pmarks98** [comment]
@mrachamallu I addressed your changes and cleaned up all the files. Made some changes to how the styles are applied and added many comments in the style.swift file for documentation on styling. I added in a constants.swift file and organized everything.

**Comment by mrachamallu** [comment]
looks good. from what i've seen, the styles file is meant for constants only, and each different component, like a button, has it's own file. we can rearrange these things later in a different pr. feel free to merge @pmarks98

**Pull request merged**